models/account_move.py

```diff
@@ -17,18 +17,21 @@ class AccountMove(models.Model):
     travaux_termines = fields.Boolean()
     salesperson_eval = fields.Boolean(string="Exclure de l'évaluation du vendeur",tracking=True)
     salesperson_eval_reason = fields.Char(string='Raison', tracking=True)
-    derniere_date_paiement = fields.Date(compute='_compute_derniere_date_paiement')
+    date_last_payment = fields.Date(compute='_compute_date_last_payment', store=True)
```

Comment:
- Kindly give  the correct name to the field . It will resolve the issue that occurs when we try to install the custo module. either use `date_last_payment` or `derniere_date_paiement`, but not both at once, and fix this in all files   in views, models, etc. :+1:  :+1:  :+1:  :+1: